### ISSUE TITLE
Literature Review: Fitness Components

### ISSUE THREAD

**Issue by donnadonwhite** [donnadonwhite/careers-in-finance#2]
FITNESSGRAM is software that was developed by Cooper institute and launched in 1982 with the aim of measuring health-related physical-related fitness. Initially, it was designed by Charles L. Sterling as a simple report care would be used to systematically measure, record and store and record information regarding subjects’ physical activity. Later on, it was taken through a series of improvements and currently, it is being used as an educational assessment tool as well as an essential reporting software program rather [how to run plagiarism check on google docs](https://plagiarismsearch.com/how-to-use-plagiarismsearch-google-add-on.html). This improvement was effected after the software was taken through physiological, behavioral, epidemiological, and pedagogical research Consequently, criterion referenced standards were developed in relation to health-related physical fitness. At this stage, the software was entirely dependent on the comparison among the subjects.
The initial emphasis of FITNESSGRAM was entirely directed towards physical activity and essential elements related to its measurement. Later on, behavioral-based recognitions were integrated to determine the relationship between behavior and the rate of physical activity. This was an important milestone that not only regarded physical activity in isolation but also considered the role of mind, mental and psychological/emotional processes in influencing physical and psychomotor processes. These latest adjustments made FITNESSGRAM a non-competitive fitness assessment for the health-related attributes. This improvement was also as a result of the Healthy Fitness Zone standards that were established in regards to the scientific procedures and norms. The software has now been unanimously adopted by schools. It is widely used by schools to evaluate as well as evolve their programs related to physical education with the ultimate aim of supporting a child in a holistic manner.
Based on the history of the evolution of FITNESSGRAM that has lasted for 36 years coupled with continuous research, validation, evaluation, and enhancement, the Scientific Advisory Board now regards FITNESSGRAM the most trusted and widely-applied educational fitness assessment and reporting tool.
Aerobic Capacity
One Mile Walk Test aims at testing the participants’ ability to complete a one-mile race within the shortest time possible. This means that the requirement to complete the race within the specified period of time is a criterion that must be adhered to any time the test is being carried out. According to Judge & Zapata, the use of criterion measure is an important requirement of both concurrent and predictive validity. While administering the test, the criterion is also collected at the same time. According to Gardner predictive validity is done to collect information for the purposes of initial decision-making. Therefore, One Mile Walk Test is justified as a reliable test based on its predictive reliability.
The healthy fitness zone for the aerobic capacity that relies on One Mile Walk Test is a component of the AAHPERD Functional Fitness Protocol which was developed purposely for the subjects of the age ranging from 13 years and older. However, it does not involve younger children below the age of 13 and also excludes old age individuals. The earlier the subject completes a one mile walking distance, the higher aerobic capacity they are. According to Outermans et al., an individual is expected to take around 20 minutes to complete the walk. However, standard deviations of +2 or -2 are permitted. According to Outermans et al., the allowance is to cater for individual differences among the participants. Generally, majority of the initially tested subjects completed the walking race at the twentieth minute.
In a study done to investigate the validity of Walk Scores and the Transit Scores from a Walk Score website, the importance of One Mile Walk Test as a valid and reliable test was established. The study was done by Duncan, Aldstadt, Whalen and Melly (2013) using 1,292 youths obtained from the Boston’s 2008 Youth Survey Geospatial Dataset based on both 400-m and 800-m street network buffers. Findings indicated that the 800-m buffer recorded all significant correlations whereas 400-m buffer GIS Measures recorded several significant correlations. Further in the findings of the study, the use of different types of walk scores, including One Mile Walk Test is a good way of measuring neighborhood workability. However, this works best with larger groups.
Portable Bioelectric Impendence Analyzers
This is a convenient approach used to investigate body composition and the results obtained are used as indicators of leanness of fatness. The method involves exposing subjects to radiation to find out the nature of resistance of various body components fat or lean components of the body are subjected to an electric current. By determining to total body water composition, the fat composition can similarly be obtained. In terms of validity, this method is concurrently valid because of its use of an impendancemetry-specific equation used for the determination of obesity. Results that go beyond or below these standards are considered supernormal and subnormal respectively. Therefore,  is the criterion-referenced standards upon which all other measurements are based. The amount of VAT and SAT have also been recently measured using this method and have indicated that the correlation is significant compared with precise imaging techniques such as CT. This further indicates a high level of validity and reliability of BIA.
A recent cross-sectional study was conducted to investigate the validity of two portable BIA devices, the Tanita BC-418  and the Inbody 230  in Taiwan. The study participants were the Taiwanese Children. Findings indicated a strong linier correlation between the two types of body composition measurements . For all the subjects of this experiment concluded that LBM measures were all accurate and precise using the BIA8MF. However, results obtained using %BF and FM recorded significant errors. This is affirmation that BIA is generally a valid and reliable method of measuring body composition.
Flexed Arm Hang Testing
Flexed Arm Hang testing is mostly employed in the military to test the strength of the upper body and endurance by timing the duration that a person takes hanging with their chin above a bar. The “FITNESSGRAM/ACTIVITYGRAM (FG) which is software applied in educational assessment as well as reporting highly recommends the use of Flexed Arm Hang Testing and in response, many schools all over the world are using this method with millions of students. Clemons  reports the method as having remained consistent in yielding reliable outcomes overtime. This consistency is a strong indicator of validity and reliability of the Flexed Arm Hang Testing. Clemons adds that FAH testing has a high level of construct validity since its dependency has firmly been established. It is on this basis that the method is unanimously depended upon among a large number of institutions and administrators.
The criterion-referenced standards of the flexed hang test are the requirement that the subjects must hang for at least fifteen seconds continuously. This is an average standard considered more than 50 percent but not the optimum standard. For the subjects who are capable of hanging for at least 70 seconds, maximum marks can be awarded to them during the testing. Timing is done on the bar and the counting of the timer is stopped the moment the chin falls below or touches the bar. Evidently, FAH is a relatively open-ended approach to the testing of muscular endurance and strength that accommodate different abilities of individuals and ranks them accordingly on a scale. In both Clemons and Bartlett, Phillips & Galarneau,  studies, there is a correlation in the results obtained from the participant’s endurance and strength, and the actual performance of the participants who were ranked higher in terms of the test outcomes. Marinho et. al.  also found from a study that the level of endurance and strength are strongly related to other body factors such as weight, age, and height. However, Marinho et al. posits that the level of endurance is at the optimum for the optimal weight, age, and height which are neither the least nor the largest. These combined findings assert the both concurrent and predictive validity of FAH.
Conclusion
In summary, various fitness components that can be measured using the above discussed test items include aerobic capacity, body composition, muscular endurance and strength. These are important indicators of health and well-being, and encompass aspects of fitness, nutritional wellness, and performance capability. However, very little has been done to identify and establish the effectiveness of tools and procedures used to measure the correlation between body functioning and mind. Therefore, future research should focus on establishing and ranking various tools and procedures to investigate bodily functions in relation to mental capability.
